app/services/analytics_events.rb

```diff
@@ -2941,6 +2942,7 @@ def idv_phone_confirmation_otp_submitted(
     errors:,
     code_expired:,
     code_matches:,
+    delivery_method:,
```

Comment:
In other events (e.g. "Multi-Factor Authentication: phone setup"), we call this property `otp_delivery_preference`, wonder if there'd be benefit to being consistent.

Reply:
Good catch - see #10444